recipe/post-link.sh

```diff
@@ -0,0 +1 @@
+"${PREFIX}/bin/python" -c 'import pysr; pysr.install()' >> "${PREFIX}/.messages.txt" 2>&1
```

Comment:
Where would this install the julia packages?

Reply:
@ngam I'm assuming the packages get installed in a location according to [this env variable](https://github.com/conda-forge/julia-feedstock/blob/b87f21cda4539b3069a369ca28e3379054eacdd1/recipe/scripts/activate.sh#L6), right?

Reply:
Oops, messages crossed :)

Reply:
If a downstream package uses pysr when being built by conda-build, will conda-build package the julia packages installed by this post-link script?

Reply:
I thought `post-link.sh` runs on the user side?

Reply:
I think we can defend against this by checking whether [`CONDA_BUILD_STATE`](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#dynamic-behavior-based-on-state-of-build-process) is not `test` in the `post-link` script.